webapp/components/features/FollowList/FollowList.vue

```diff
@@ -0,0 +1,165 @@
+<template>
+  <base-card class="follow-list">
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-26T18:04:17Z" title="Thursday, March 26th 2020, 7:04:17 pm +01:00">Mar 26, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,157 @@
+<template>
+  <base-card class="hc-connections">
```


```suggestion
  <base-card class="follow-list">
```
we give the top level `class` in our components the same name as the component.
we should probably document this, but we can do that in another `PR`